includes/nginx.conf

```diff
@@ -2,6 +2,21 @@
 # openHABian NGINX Configuration #
 ##################################
 
+
+## Forward proxy
+## Configure <server>:8888 as a manual (forward) proxy in your browser
+## Access any local device in a *remote* (Tailscale VPN) location by target IP
+
```

Comment:
Is this the only change we need to make other than what you mentioned in task 2 on the issue? Also please tell me this doesn't allow remote access to any device on the RPi's LAN as that would be a big security issue.

Reply:
Yes it's the only change.
Proxy is on port 8888 which of course noone should publish/open to the outside.
Yes it allows for access to any device - but that is just like you can access these when you're in the LAN.
So I don't see the security issue in here ?
Maybe we should restrict it to *web* interface, but thing is it would need to be ports 80 AND 443 and I don't know how to configure that right now other than to allow on all ports.  Also, nginx is not installed by default.

Reply:
Fair enough, I just see the potential for malicious actors to find a way into a LAN if given the chance and from there they could potentially do some nasty stuff.